db/init_db.js

```diff
@@ -11,6 +11,19 @@ async function buildTables() {
     // drop tables in correct order
 
     // build tables in correct order
+    await client.query(`
+      CREATE TABLE users (
+        id SERIAL PRIMARY key,
+        firstName VARCHAR(255) NOT NULL,
+        lastName TEXT NOT NULL,
+        email VARCAHAR(320) NOT NULL,
+        image VARCHAR(255) UNIQUE,
```

Comment:
image should be imageURL
this also needs a default value, you can do something like `imageURL text default 'someone@gmail.com'`